docs/hydra/self-hosted/migrating-from-mitreid.mdx

```diff
@@ -5,26 +5,26 @@ title: Migrating from MITREid
 
 :::info
 
-Please note this page doesn't pretend to be a step-by-step guide.
+Please note this page doesn't intend to be a step-by-step guide.
 
 :::
 
-This piece of documentation will try to give you some hints to move your current MITREid instance to Hydra.
+This guide will try to give you some hints on moving your current MITREid instance to Hydra.
 
-#### Scopes
+# Scopes
 
-Scopes in hydra doesn't have a relational entity, every client defines its own array of scopes.
+Scopes in hydra do not have a relational entity, every client defines its own array of scopes.
 
-#### Scope vs Scp
+# Scope vs Scp
```

Comment:
Please don't use H1 excessively.

https://contentmarketinginstitute.com/articles/how-tags-matter-seo/


/cc @vinckr please check this when reviewing.

Reply:
No need to update this, I'm doing this in the refactor.

Reply:
Ah, true. We should have a linter for that. Thanks.

Reply:
my bad i missed it in the diff 🙈